common.hpp

```diff
@@ -57,7 +63,7 @@ static int compare_buf(uint32_t *ref_buf, uint32_t *buf, size_t size)
 {
     int diff = 0;
     if (size % 4 != 0U) {
-        printf("warning: buffer size %d is not dword aligned, ignoring trailing bytes\n");
+        printf("warning: buffer size %d is not dword aligned, ignoring trailing bytes\n", size);
```

Comment:
Suggest to use %zu instead of %d because the type is "size_t"

Reply:
good point, done!